index.js

```diff
@@ -63,6 +73,14 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+let weather = 'Rainy';
+let message;
+if (weather = 'Sunny') {
```

Comment:
when you type a single `=` sign, javasript understands that as the keyword for assigning a value to a variable (like `let firstName = 'Hassan'`, so on line 78 you actually want to use `===` to compare values.